Jenkinsfile

```diff
@@ -120,9 +120,9 @@ pipeline {
         stage('cmake-jammy-gcc') {
           agent {
             dockerfile {
-              additionalBuildArgs '--pull'
-              dir '.CI/cache'
+              image 'AnHeuermann/build-deps:v1.22.1-qtwebengine'
               label 'linux'
+              alwaysPull true
               args "--mount type=volume,source=omlibrary-cache,target=/cache/omlibrary " +
                    "-v /var/lib/jenkins/gitcache:/var/lib/jenkins/gitcache"
             }
```

Comment:
This can't be `dockerfile {` if you specify the image directly. It needs to be `docker {`.